app/views/past_events/_form.html.erb

```diff
@@ -0,0 +1,32 @@
+<%= form_with(model: past_event, local: true) do |form| %>
+  <% if past_event.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(past_event.errors.count, "error") %> prohibited this past_event from being saved:</h2>
+
+      <ul>
+      <% past_event.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <%= form.label :title, "Title" %> <br>
+  <%= form.text_field :title, :required => true%> <br>
+
+  <%= form.label :subtitle, "Subtitle" %> <br>
+  <%= form.text_field :subtitle %> <br>
+
+  <%= form.label :description, "Description" %> <br>
+  <%= form.text_area :description %> <br>
+
+  <%= form.label :year, "Year" %> <br>
+  <%= form.number_field :year, :required => true%> <br>
```

Comment:
can add minimum and maximum values for the number field

Reply:
I will fix this when I am doing validation